### PR TITLE
📝 [Documentation] Improve doc theme and update README

### PR DESCRIPTION
**Pull request by bruhnild**
## Checklist

- [ ] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/contribute/contributing.html)
- [ ] My code respects the Definition of done available in the [Development section of the documentation](https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields)
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes
- [x] I added an entry in the changelog file
- [x] My commits are all using prefix convention (emoji + tag name) and references associated issues
- [ ] I added a label to the PR corresponding to the perimeter of my contribution
- [ ] The title of my PR mentionned the issue associated

### CODE REVIEW

**Comment by camillemonchicourt** [comment]
Je passerai bien toute la doc de RST à MD partout.
Par contre commencer à mettre du HTML, je suis moins chaud, mais si c'est utile, pourquoi pas.

File: README.rst [hunk 1]
```diff
@@ -1,112 +1,138 @@
...
+
+**Geotrek-admin** is a web application designed to manage, centralize, and structure geographical and touristic information for your territory. It is the back-office application of the Geotrek ecosystem.
+
+With Geotrek-admin, you can:
+
+- Manage treks, touristic information, and related content (media, descriptions, etc.).
+- Organize your data with maps, layers, and metadata.
+- Export content to various public interfaces, such as Geotrek-rando or printed topoguides.
+
+You can explore Geotrek-admin in action through our demonstration website:
```
Comment (camillemonchicourt) [inline README.rst:49]:
```suggestion
You can explore Geotrek-admin in action through the demonstration website:
```

File: README.rst [hunk 2]
```diff
@@ -1,112 +1,138 @@
...
 
 .. image:: https://geotrek.fr/assets/img/logo_autonomens-h120m.png
-    :target: https://datatheca.com
+    :alt: Logo Autonomens
+    :target: https://datatheca.com/
 
 ----
 
 .. image:: http://geotrek.fr/assets/img/parc_ecrins.png
+    :alt: Parc des Ecrins
```
Comment (camillemonchicourt) [inline README.rst:129]:
```suggestion
    :alt: Parc nartional des Ecrins
```

File: README.rst [hunk 3]
```diff
@@ -1,112 +1,138 @@
...
     :target: http://www.ecrins-parcnational.fr
 
-
 .. image:: http://geotrek.fr/assets/img/parc_mercantour.png
+    :alt: Parc du Mercantour
```
Comment (camillemonchicourt) [inline README.rst:133]:
```suggestion
    :alt: Parc national du Mercantour
```

File: README.rst [hunk 4]
```diff
@@ -1,112 +1,138 @@
...
+    :target: https://github.com/GeotrekCE/Geotrek-admin/graphs/contributors
+
+Made with `contrib.rocks <https://contrib.rocks>`_.
 
 License
 -------
 
-* OpenSource - BSD
-* Copyright (c) 2012-2023 - Makina Corpus / Parc national des Ecrins - Parc National du Mercantour - Parco delle Alpi Marittime
+This project is under the MIT License. See the `LICENSE <Geotrek-admin/blob/main/LICENSE>`_ for details.
```
Comment (camillemonchicourt) [inline README.rst:112]: GTA est en BSD, pas MIT

**Review by submarcos — APPROVED** [review]

**Review by camillemonchicourt — APPROVED** [review]

**Pull request merged**